### ISSUE TITLE
Other motions from Trello

### ISSUE THREAD

**Issue by nonprofittechy** [LemmaLegalConsulting/docassemble-MOHUDEvictionProject#303]

**Comment by miabonardi** [comment]
Should we ask Terry for drafts of these motions?

- Motion to Quash/Stay Eviction
- Request for de novo Hearing
- Motion to dismiss for lack of service

**Comment by nonprofittechy** [comment]
Yes, let's do that. Also they can be for real tenants, we can do work to
generalize them
